ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master__ocp-4.5.yaml

```diff
@@ -0,0 +1,37 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+raw_steps:
+- pipeline_image_cache_step:
+    commands: |
+      yum install -y epel-release 
+      yum install -y ansible python-pip
+      pip install packet-python
+      chmod g+w /etc/passwd
+    from: src
+    to: devscripts-base
```

Comment:
What's the benefit of doing all of these steps in this YAML, instead of having it in the Dockerfile by default? 

I could imagine eventually moving to a container-based use of dev-scripts even for developers running it on their own box

Reply:
It's an approach that I found as temporary workaround to avoid working on two repos at the same time, since the Dockerfile is located into dev-scripts. It adds more flexibility in doing some tests and, as reported in https://github.com/openshift-metal3/dev-scripts/pull/1024, once consolidated it be moved into the Dockerfile (another alternative approach could have been to use WIP config pointing to a dev-script fork)